app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx

```diff
@@ -234,6 +234,7 @@ export const FormAudience = ({
                 type="number"
                 min="0"
                 max="100"
+                step="0.01"
```

Comment:
populationPercent allows 4 decimal places:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/models/nimbus.py#L47-L49

Reply:
I don't think this restricts the number of decimal places, just indicates that it's not an integer and provides a step value for the up/down arrows